sdk/python/feast/expediagroup/vectordb/elasticsearch_online_store.py

```diff
@@ -205,3 +252,31 @@ def _get_value_from_value_proto(self, proto: ValueProto):
             value = list(value.val)
 
         return value
+
+    def _create_value_proto(self, feature_val, value_type) -> ValueProto:
```

Comment:
Let's create a common utility method if this method is the same or similar to what already exists for the Milvus implementation

Reply:
It is slightly different since Milvus and ES handle certain types like the timestamp in different ways. Actually I just realized it doesn't cover all cases, I'll fix that.